### ISSUE TITLE
Making the height of iframe to fit the content

### ISSUE THREAD

**Issue by nsunami** [eur-nl/erim-rdmkit#3]
Currently, the iframe is rendering on the ERIM test website has a fixed height, and thus sometimes the content overflow vertically. This requires scrolling on users' side, which is not convenient.

Instead, I want to make the height of the iframe responsive to the content rendered on RDMkit.

Using JS on the parent page is not possible because of the cross-origin issue.

Instead, I want to try out the solution described in this [Stack Overflow post](https://stackoverflow.com/a/6940531/6205282).

In a nutshell:

1. use `parent.postMessage` in the RDMkit page to send the `document.body.scrollHeight` to the parent
2. add an event listener on the parent page to resize the height of iframe

**Comment by nsunami** [comment]
The `parent.postMessage` method did not work. Instead, I'm trying out [iframeResizer](https://github.com/davidjbradshaw/iframe-resizer) instead.
